public/index.html

```diff
@@ -26,6 +26,10 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
 
+    <!-- Cloudflare Web Analytics -->
+    <script defer src='https://static.cloudflareinsights.com/beacon.min.js' data-cf-beacon='{"token": "fb6f5378f9fa492e9b989811ee2b9323"}'></script>
```

Comment:
@lacabra is the `token` safe to be put here and not in an env variable?

Reply:
Yes, because it's on the frontend anyway, and thus always executed client-side. It's analogous to the Google Analytics ID. I'm fairly certain that Cloudflare then matches token with origin to validate it.